Patches/Rim-Sheks!/Patch_PawnKinds_Sheks.xml

```diff
@@ -0,0 +1,193 @@
+								</li>
+							</sidearms>
+						</li>
+					</value>
+				</li>
+  </li> -->
+
+				<!-- SHEK Pirates-->
+
+				<li Class="PatchOperationAddModExtension">
+					<xpath>/Defs/PawnKindDef[defName="Scavenger_Shek"]</xpath>
+					<value>
+						<li Class="CombatExtended.LoadoutPropertiesExtension">
+							<primaryMagazineCount>
+								<min>2</min>
+								<max>3</max>
+							</primaryMagazineCount>
+							<sidearms>
+								<li>
+									<sidearmMoney>
+										<min>40</min>
+										<max>160</max>
+									</sidearmMoney>
+									<weaponTags>
+										<li>CE_Sidearm_Tribal</li>
+										<li>CE_Sidearm_Melee</li>
+									</weaponTags>
+								</li>
+							</sidearms>
+						</li>
+					</value>
+				</li>
+
+				<!-- SHEK WARRIOR -->
+				<li Class="PatchOperationAddModExtension">
+					<xpath>/Defs/PawnKindDef[defName="Shek_Warrior" or defName="Shek_Merchant" or defName="Shek_Vassal"]</xpath>
+					<value>
+						<li Class="CombatExtended.LoadoutPropertiesExtension">
+							<primaryMagazineCount>
+								<min>2</min>
+								<max>3</max>
+							</primaryMagazineCount>
+							<shieldMoney>
+								<min>100</min>
+								<max>150</max>
+							</shieldMoney>
+							<shieldTags>
+								<li>TribalShield</li>
+							</shieldTags>
+							<shieldChance>0.5</shieldChance>
+							<sidearms>
+								<li>
+									<sidearmMoney>
+										<min>40</min>
+										<max>160</max>
+									</sidearmMoney>
+									<weaponTags>
+										<li>CE_Sidearm_Tribal</li>
+										<li>CE_Sidearm_Melee</li>
+									</weaponTags>
+								</li>
+							</sidearms>
+						</li>
+					</value>
+				</li>
+
+				<!-- SHEK PYRO -->
+				<li Class="PatchOperationAddModExtension">
+					<xpath>/Defs/PawnKindDef[defName="Shek_Pyrotechnician"]</xpath>
+					<value>
+						<li Class="CombatExtended.LoadoutPropertiesExtension">
+							<primaryMagazineCount>
+								<min>10</min>
+								<max>22</max>
+							</primaryMagazineCount>
+							<shieldMoney>
+								<min>100</min>
+								<max>150</max>
+							</shieldMoney>
+							<shieldTags>
+								<li>TribalShield</li>
+							</shieldTags>
+							<shieldChance>0.5</shieldChance>
+							<sidearms>
+								<li>
+									<sidearmMoney>
+										<min>40</min>
+										<max>160</max>
+									</sidearmMoney>
+									<weaponTags>
+										<li>CE_Sidearm_Melee</li>
+									</weaponTags>
+								</li>
+							</sidearms>
+						</li>
+					</value>
+				</li>
+
+				<!-- SHEK GUNNERS -->
+				<li Class="PatchOperationAddModExtension">
+					<xpath>/Defs/PawnKindDef[defName="Shek_Guardian" or defName="Heavy_Shek_Guardian"]</xpath>
+					<value>
+						<li Class="CombatExtended.LoadoutPropertiesExtension">
+							<primaryMagazineCount>
+								<min>3</min>
+								<max>4</max>
+							</primaryMagazineCount>
+							<shieldMoney>
+								<min>100</min>
+								<max>150</max>
+							</shieldMoney>
+							<shieldTags>
+								<li>TribalShield</li>
```

Comment:
These pawns are equipped with outlander-level equipment. You could potentially give them a small chance at a modern ballistic shield, but a tribal shield is probably more of a hinderance than it's worth to these pawns.

Reply:
Not sure if I want to give them outlander shields, outlander shields are pretty OP(there's a reason why no shield ever is made from a gigantic 8mm thick steel plate) and makes things too tanky.

If you ask me, outlander shields should be reworked into using primarily composite material, but that's another issue.

Reply:
Come to think of it, I should add the nose tag to humanoid noses and redo the patches to certain apparel so that the eyes aren't covered.
This way we won't have helmets with gigantic and obvious eye slits protecting the eyes as well.

Reply:
If I have your blessing, I'll make the aforementioned changes(nose and eye coverage changes). I would hate to have spent all the time doing something only to have it sit in the PR forever.

Reply:
Hell, I'll make add the left arm/hand and right arm/hand tag as well, so shield wouldn't really protect your weapon hand.

Reply:
If you'd like, you can put together a PR, we can discuss it further there. The alternative may just be to add additional ingredients so the shield can be justified to have a ballistic glass insert, as a ballistic shield would.